docs/soci.md

```diff
@@ -0,0 +1,29 @@
+# Lazy-pulling using SOCI Snapshotter
+
+| :zap: Requirement | nerdctl >= 1.1.1 |
+| ----------------- |------------------|
+
+SOCI Snapshotter is a containerd snapshotter plugin. It enables standard OCI images to be lazily loaded without requiring a build-time conversion step. "SOCI" is short for "Seekable OCI", and is pronounced "so-CHEE".
+
+See https://github.com/awslabs/soci-snapshotter to learn further information.
+
+## Enable SOCI for `nerdctl run`
+
+- Install containerd remote snapshotter plugin (`soci`) from https://github.com/awslabs/soci-snapshotter/blob/main/docs/GETTING_STARTED.md#prerequisites
+
+- Add the following to `/etc/containerd/config.toml`:
+```toml
+[proxy_plugins]
+  [proxy_plugins.soci]
+    type = "snapshot"
+    address = "/run/soci-snapshotter-grpc/soci-snapshotter-grpc.sock"
+```
+
+- Launch `containerd` and `soci-snapshotter`
+
+- Run `nerdctl` with `--snapshotter=soci`
+```console
+nerdctl run --it --rm --snapshotter=soci docker.io/library/alpine:3.14.2
```

Comment:
In order for users to get the full benefit of using soci the image being pulled has to have a soci index present in the repository. So, the example should use an image with an already created index (i.e. https://gallery.ecr.aws/soci-workshop-examples/ffmpeg?page=1)

```suggestion
nerdctl run --it --rm --snapshotter=soci public.ecr.aws/soci-workshop-examples/ffmpeg:latest
```